helm/single-content/templates/ingress.yaml

```diff
@@ -21,14 +21,14 @@ spec:
   rules:
   - http:
       paths:
-      - path: {{ include "idol-library.ingress.path" (dict "ingress" $component.ingress "path" "/content/") }}
+      - path: {{ include "idol-library.ingress.path" (dict "ingress" $component.ingress "path" $component.ingress.path) }}
         pathType: {{ include "idol-library.ingress.pathtype" $component }}
         backend:
           service:
             name:  {{ $component.name | quote }}
             port:
               name: aci-port
-      - path: {{ include "idol-library.ingress.path" (dict "ingress" $component.ingress "path" "/index/") }}
+      - path: {{ include "idol-library.ingress.path" (dict "ingress" $component.ingress "path" $component.ingress.indexPath) }}
```

Comment:
If we use `isAgentstore: true` then maybe use the `.Values.name` to generate the paths here? So e.g. with `.Values.name==idol-agentstore` then ingress paths could be `/idol-agentstore` and `/idol-agentstore-index`